pyg_lib/csrc/sampler/dist_neighbor.cpp

```diff
@@ -0,0 +1,86 @@
+#include "dist_neighbor.h"
+
+#include <ATen/core/dispatch/Dispatcher.h>
+#include <torch/library.h>
+
+#include "pyg_lib/csrc/utils/check.h"
+
+namespace pyg {
+namespace sampler {
+
+std::tuple<at::Tensor, at::Tensor> relabel_neighborhood(
+    const at::Tensor& seed,
+    const at::Tensor& sampled_nodes_with_dupl,
+    const std::vector<int64_t>& sampled_nbrs_per_node,
+    const int64_t num_nodes,
+    const c10::optional<at::Tensor>& batch,
+    bool csc,
+    bool disjoint) {
+  at::TensorArg seed_t{seed, "seed", 1};
+  at::TensorArg sampled_nodes_with_dupl_t{sampled_nodes_with_dupl,
```

Comment:
This name suggests it is an alias to a type. A better option would be `_v` or `_arg` as a suffix.